### PR TITLE
Update networks.mdx

### PR DESCRIPTION
**Pull request by maradeeym**
Added dRPC to the list.

dRPC is decentralized RPC node provider for Evmos.

### CODE REVIEW

**Review by 0xstepit — APPROVED** [review]
Hey @maradeeym thanks for adding these endpoints!

File: docs/develop/api/networks.mdx [hunk 1]
```diff
@@ -86,6 +89,9 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://evmos-trpc.antrixy.org`             | `Tendermint` `RPC`     | Antrix Validators                       | Pruned  |
 | `https://evmos-trest.antrixy.org`            | `Cosmos` `REST`        | Antrix Validators                       | Pruned  |
 | `https://evmos-tgrpc.antrixy.org`            | `Cosmos` `gRPC`        | Antrix Validators                       | Pruned  |
+| `https://evmos-testnet.drpc.org`             | `Ethereum` `JSON-RPC`  | [dRPC](https://drpc.org/)               | Pruned  |
```
Comment (0xstepit) [inline docs/develop/api/networks.mdx:92]: This is pointing to the mainnet page. It should be:

```suggestion
| `https://drpc.org/chainlist/evmos#evmos-testnet`             | `Ethereum` `JSON-RPC`  | [dRPC](https://drpc.org/)               | Pruned  |
```

File: docs/develop/api/networks.mdx [hunk 2]
```diff
@@ -62,6 +62,9 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://evmos-grpc.antrixy.org`              | `Cosmos` `gRPC`        | Antrix Validators                               | Pruned  |
 | `https://evmos-rpc.antrixy.org`               | `Tendermint` `RPC`     | Antrix Validators                               | Pruned  |
 | `https://evmos-json.antrixy.org`              | `Ethereum` `JSON-RPC`  | Antrix Validators                               | Pruned  |
+| `https://evmos.drpc.org`                      | Èthereum` `JSON-RPC`   | [dRPC](https://drpc.org/)  | Pruned |
```
Comment (MalteHerrmann) [inline docs/develop/api/networks.mdx:65]:
```suggestion
| `https://evmos.drpc.org`                      | `Ethereum` `JSON-RPC`   | [dRPC](https://drpc.org/)  | Pruned |
```

File: docs/develop/api/networks.mdx [hunk 3]
```diff
@@ -86,6 +89,9 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://evmos-trpc.antrixy.org`             | `Tendermint` `RPC`     | Antrix Validators                       | Pruned  |
 | `https://evmos-trest.antrixy.org`            | `Cosmos` `REST`        | Antrix Validators                       | Pruned  |
 | `https://evmos-tgrpc.antrixy.org`            | `Cosmos` `gRPC`        | Antrix Validators                       | Pruned  |
+| `https://drpc.org/chainlist/evmos#evmos-testnet`             | `Ethereum` `JSON-RPC`  | [dRPC](https://drpc.org/)               | Pruned  |
```
Comment (MalteHerrmann) [inline docs/develop/api/networks.mdx:92]: from this website I'm taking that the actual endpoint is rather this one?

```suggestion
| `https://evmos-testnet.drpc.org`             | `Ethereum` `JSON-RPC`  | [dRPC](https://drpc.org/)               | Pruned  |
```

**Review by MalteHerrmann — APPROVED** [review]
Thanks for adding this 🚀 left two suggestions

**Pull request merged**